### ISSUE TITLE
Позиции игроков в играх 4 на 1 должны быть случайны

### ISSUE THREAD

**Issue by brunneng** [MailRuChamps/raic-2020#45]
Во всех этих играх, LeeTik сбоку против часовой стрелки от GreenTea
Romka всегда напротив в углу от GreenTea

https://russianaicup.ru/game/view/10625
https://russianaicup.ru/game/view/10626
https://russianaicup.ru/game/view/10627
https://russianaicup.ru/game/view/10748
https://russianaicup.ru/game/view/10690

относительные позиции довольно сильно влияют на исход и они должны быть случайны.

**Comment by adler3d** [comment]
вроде сейчас/раньше если создаешь игру сам, то можно задать кто где должен быть:  
1) снизу-слева  
2) сверху-справа  
3) снизу-справа
4) сверху-слева

очень удобно это использовать если создаёшь игры скриптом.

для желающих(можно даже используя userscript самому для себя сделать) на сайте при создание игры можно просто добавить кнопку "перемешать игроков".

**Comment by brunneng** [comment]
Не уверен что это работает. Игры создавались разными людьми, вряд ли они так специально расставили одинаково.

**Comment by oPOCCOMAXAo** [comment]
Вероятнее всего игры были скопированы, поэтому расположение скопировалось так же

**Comment by oPOCCOMAXAo** [comment]
Вот даже возможный кандидат в оригиналы: https://russianaicup.ru/game/view/10595

**Comment by brunneng** [comment]
Нет, я не могу найти ни одной системной игры, где бы Ромка был не в противоположном углу от GreenTea
https://russianaicup.ru/game/view/15379
https://russianaicup.ru/game/view/15317
https://russianaicup.ru/game/view/15246
https://russianaicup.ru/game/view/14280

**Comment by AndrewB330** [comment]
https://russianaicup.ru/game/view/16021 ?
Или имеются ввиду системные игры?

**Comment by boriszaitsev** [comment]
Если схема формирования игры такая, что как-то берутся стратегии из списка, и им отдаются позиции в  порядке рейтинга, то есть сильнейший всегда занимает позицию 1 в игре, второй по рейтингу занимает позицию 2 и так далее, то это беда и дисбаланс. 
Первая четверть игры по сути командная, диагональ против диагонали, выходит что сначала играет сильная пара против слабой.
Я посмотрел реплеи лидеров, открытых системой, первую и вторую страницу, похоже что это всегда так.
Нужно делать шафл перед распределением позиций.
